### PR TITLE
chore: add volunteer vasujhawar2001

### PR DESCRIPTION
**Pull request by vasujhawar2001**
None.
Let me know if any changes, and love this project.

### CODE REVIEW

File: data/users/vasujhawar2001.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Vasu Jhawar",
+  "bio": "Software Engineer",
+  "image": "https://pbs.twimg.com/profile_images/1485515284526297090/kL9h_WcB_400x400.jpg",
```
Comment (codewithvoid) [inline data/users/vasujhawar2001.json:4]: can you use github image url here

File: data/users/vasujhawar2001.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["react.js", "next.js", "mongoDB"]
+    },
+    {
+      "slug": "software-development",
+      "topics": ["javascript", "pyhton", "TypeScript", "Java"]
```
Comment (codewithvoid) [inline data/users/vasujhawar2001.json:12]: - use `programming` slug
- limit topics to max 2-3
- fix typo in python language

File: data/users/vasujhawar2001.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+      "slug": "Computer Science",
+      "topics": ["Data Structures", "Algorithms", "Systems Design"]
```
Comment (codewithvoid) [inline data/users/vasujhawar2001.json:16]: mention `DSA` in programming topic. mention `System Design` in `software-development` category

File: data/users/vasujhawar2001.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Vasu Jhawar",
+  "bio": "Software Engineer",
+  "image": "https://pbs.twimg.com/profile_images/1485515284526297090/kL9h_WcB_400x400.jpg",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["react.js", "next.js", "mongoDB"]
```
Comment (codewithvoid) [inline data/users/vasujhawar2001.json:8]: mention MongoDB in `database-development` category slug

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @vasujhawar2001 for volunteering. Added some comments.

File: data/users/vasujhawar2001.json (hunk 1 above)
Comment (vasujhawar2001) [reply data/users/vasujhawar2001.json]: done!

**Review by codewithvoid — APPROVED** [review]
Thanks @vasujhawar2001 for making the changes.

**Pull request merged**